yt_dlp/extractor/vrt.py

```diff
@@ -425,3 +427,74 @@ def _real_extract(self, url):
                 ['description', 'twitter:description', 'og:description'], webpage),
             '_old_archive_ids': [make_archive_id('Canvas', video_id)],
         }
+
+
+class Radio1BeIE(VRTBaseIE):
+    _VALID_URL = r'https?://radio1\.be/\w+(/[\w-]+/[\w-]+)?/(?P<display_id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://radio1.be/luister/select/de-ochtend/komt-n-va-volgend-jaar-op-in-wallonie',
+        'info_dict': {
+            'id': 'eb6c22e9-544f-44f4-af39-cf8cccd29e22',
+            'title': 'Komt N-VA volgend jaar op in Wallonië?',
+            'display_id': 'komt-n-va-volgend-jaar-op-in-wallonie',
+            'description': 'md5:b374ea1c9302f38362df9dea1931468e',
+            'thumbnail': r're:https?://cds\.vrt\.radio/[^/#\?&]+'
+        },
+        'playlist_mincount': 1
+    }, {
+        'url': 'https://radio1.be/lees/europese-unie-wil-onmiddellijke-humanitaire-pauze-en-duurzaam-staakt-het-vuren-in-gaza?view=web',
+        'info_dict': {
+            'id': '5d47f102-dbdb-4fa0-832b-26c1870311f2',
+            'title': 'Europese Unie wil "onmiddellijke humanitaire pauze" en "duurzaam staakt-het-vuren" in Gaza',
+            'description': 'md5:1aad1fae7d39edeffde5d3e67d276b64',
+            'thumbnail': r're:https?://cds\.vrt\.radio/[^/#\?&]+',
+            'display_id': 'europese-unie-wil-onmiddellijke-humanitaire-pauze-en-duurzaam-staakt-het-vuren-in-gaza'
+        },
+        'playlist_mincount': 1
+    }]
+
+    def _extract_video_entries(self, next_js_data, display_id):
+        entries = []
+        video_data = variadic(traverse_obj(
+            next_js_data, {lambda x: x if x.get('mediaReference') else traverse_obj(x, ('paragraphs', ...))}))
```

Comment:
We can already check that all entries have a mediaReference
```suggestion
        video_data = traverse_obj(
            next_js_data, ((None, ('paragraphs', ...)), {lambda x: x if x['mediaReference'] else None}))
```